jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ProjectAccountSettingsManager.kt

```diff
@@ -290,13 +305,6 @@ fun Project.activeRegion(): AwsRegion = ProjectAccountSettingsManager.getInstanc
  */
 fun Project.activeCredentialProvider(): ToolkitCredentialsProvider = ProjectAccountSettingsManager.getInstance(this).activeCredentialProvider
 
-/**
- * The underlying AWS account for current active credential provider of the project. Return null if credential provider is not set.
- * Calls of this member should be in non-UI thread since it makes network call using an STS client for retrieving the
- * underlying AWS account.
- */
-fun Project.activeAwsAccount(): String? = tryOrNull { AwsResourceCache.getInstance(this).getResourceNow(StsResources.ACCOUNT) }
```

Comment:
Could probably just kill `activeAwsAccountIfKnown` altogether - it's only used in the telemetry service.

Reply:
Ya, I put it here due to activeAwsAccount is here. We should move it into telemetry service itself.

Reply:
I'll do that on a PR to master

Reply:
#1810